coral/src/app/pages/topics/index.test.tsx

```diff
@@ -60,6 +64,47 @@ describe("Topics", () => {
       expect(headline).toBeVisible();
     });
 
+    it("renders 'Request A New Topic' primary action in heading", async () => {
+      const headline = screen.getByRole("button", {
+        name: "Request A New Topic",
+      });
+
+      expect(headline).toBeVisible();
+    });
+
+    it("renders 'Request A New Topic' primary action in heading", async () => {
+      const primaryAction = screen.getByRole("button", {
+        name: "Request A New Topic",
+      });
+
+      expect(primaryAction).toBeVisible();
+    });
+
+    it("navigates to '/requestTopics' on clicking the primary action in heading", async () => {
```

Comment:
Pretty unsure about my approach for this test, would be grateful for feedback ^^

Reply:
Looks good for me, I would have done it similar!

I would recommend to update the test description and at least use "clicking "Request A New Topic" button" instead of "primary action". 

Primary action is a implementation detail that does not matter for the test (or user). It can cause a moment of confusion when reading the test, so I would remove that. 

(maybe be also more user-centric like "navigates to (...) when user clicks the button (...)" but that's very nitpicky and I struggle with that myself all the time 😅 )

Reply:
Not nitpicky, it's an art to formulate good test descriptions ^^

Reply:
>Not nitpicky, it's an art to formulate good test descriptions ^^

![cat nodding saying: So true, so true](https://media.tenor.com/7gw0yQXn60oAAAAC/so-true.gif)